package.json

```diff
@@ -2,8 +2,8 @@
   "name": "react-intl-native",
   "version": "2.1.1",
   "description": "Convenience components for react-intl's `format*` API in React Native",
-  "main": "dist/react-intl-native.js",
-  "jsnext:main": "dist/react-intl-native.es2015.js",
+  "main": "src/index.js",
+  "jsnext:main": "src/index.js",
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "rollup-babel-lib-bundler -f cjs,es6 src/index.js",
```

Comment:
Since it's directly referencing the source, we don't need the rollup builds any more. I'm not sure if it's in the scope of the pull request though.